engine/lib/ast.ml

```diff
@@ -553,6 +553,7 @@ functor
           of_trait : (global_ident * generic_value list) option;
           items : impl_item list;
         }
+      | Use of string list * string * bool * string option
```

Comment:
For uniformity, I'd prefer a record payload here, i.e.
```suggestion
      | Use of { path: string list; external: bool; rename: string }
```

About the `string` field for Single, Glob, etc.: I don't really see how we could use this information. Do you have ideas in mind?

Reply:
I agree, I just did not want to throw away potential useful information. However this can be re-added if it is ever needed.